ext/RMagick/rmimage.c

```diff
@@ -13042,21 +13045,20 @@ Image_store_pixels(VALUE self, VALUE x_arg, VALUE y_arg, VALUE cols_arg
     size = (long)(cols * rows);
     rm_check_ary_len(new_pixels, size);
 
+    exception = AcquireExceptionInfo();
```

Comment:
I think this line does not need to move.
It is easier to understand when it is executed immediately before use.

Reply:
I moved it a bit earlier because this needs to be done in ImageMagick 7. The diff with ImageMagick 7 will be cleaner if this is done first.